ports/boinc/vcpkg.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "boinc",
+  "version": "7.18.1",
+  "description": "Open-source software for volunteer computing and grid computing.",
+  "homepage": "https://boinc.berkeley.edu/",
+  "supports": "!((windows & (arm | arm64)) | uwp)",
```

@AenBleidd, Could you please clarify why these are not supported?

---

UWP is not supported because of some WinAPI calls (CreateFile, CreateProcess, etc). I plan to work on upstream to define whether it's possible at all to build it for UWP.
Regarding ARM(64), upstream also has no support of this architecture for Windows and has some x86/x64 only instructions.
I also plan to work on upstream to fix that, but it's definitely not a part of vcpkg port script

---

If arm is not supported (including on OSX), this should probably be:

```suggestion
  "supports": "!arm & !uwp",
```

If ARM is only not supported on Windows, you could also do:

```suggestion
  "supports": "!(windows & arm) & !uwp",
```